src/app/routes/article/getInitialData/index.ts

```diff
@@ -91,15 +103,39 @@ export default async ({
 
     let wsojData = [];
     try {
+      // @ts-ignore - Ignore fetchPageData argument types
       const { json: wsojJson = [] } = await fetchPageData({
         path: wsojURL,
-        ...({ agent, optHeaders } as any),
+        ...{ agent, optHeaders },
       });
       wsojData = wsojJson;
     } catch (error) {
       logger.error('Recommendations JSON malformed', error);
     }
 
+    // Ensure all local CPS fixture and test data is in the correct format
+    if (isLocal && pageType === 'cpsAsset') {
+      const secondaryData = await getAdditionalPageData({
+        pageData: json,
+        service,
+        variant,
+        env,
+      });
+
+      json = {
+        data: {
+          article: json,
+          // Checks for data mocked in tests, or data from fixture data
+          secondaryData: json?.secondaryData ?? {
+            topStories: secondaryData?.secondaryColumn?.topStories,
+            features: secondaryData?.secondaryColumn?.features,
+            mostRead: secondaryData?.mostRead,
+            mostWatched: secondaryData?.mostWatched,
+          },
+        },
+      };
+    }
```

Comment:
This will be removed once we migrate fully to the BFF for CPS assets. 

This is purely to support local fixture/test data so that we don't need to update tons of fixture data and tests to account for the new BFF data structure.

A lot of tests import these fixture data JSON files directly in the tests, so would in turn require the tests themselves to be altered to point to the nested object structure that the BFF returns. This requires **_tons_** of changes, which would then need changed _again_ once we move the data transformation steps to the BFF, and then potentially _again_ once we start fetching CPS assets from the new CAF endpoint.